__tests__/original.js

```diff
@@ -0,0 +1,22 @@
+"use strict"
+import produce, {original} from "../src/immer"
+
+describe("original", () => {
+    const baseState = {
+        a: [],
+        b: {}
+    }
+
+    it("should return the original from the proxy", () => {
+        const nextState = produce(baseState, draftState => {
```

Comment:
could you update the test to check both with `useProxies(true)` and `useProxies(false)`?